React-Template-master/src/pages/SignIn.tsx

```diff
@@ -56,14 +84,18 @@ export const SignIn = (): JSX.Element => {
                             value={data.email}
                             onChange={async (e): Promise<void> => {
                                 setData({ ...data, email: e.target.value });
-                                setIsValidEmail(await emailValidation(e.target.value));
+                                if (!(await emailValidation(e.target.value))) {
```

Comment:
Store the await result in a variable before checking.

Reply:
okay